src/test/functional/steps/court-details.ts

```diff
@@ -206,6 +206,8 @@ Then( 'I click the link {string} from {string} and it takes me to the page{strin
 
   I.seeTextEquals(pageLink,'#'+ sidebarEntry +' > ul >li:nth-child('+ index +') > a');
   I.click('#'+ sidebarEntry + '> ul > li:nth-child('+ index +') > a');
+  I.wait(5);
+  I.switchToNextTab();
```

Comment:
I think maybe in a newer version of codecept where switchToNextTab has a built in parameter to wait some time for a tab to load but ours doesn't so had to use I.wait. might be able to play with timing i left it at 5 to be safe.